### PR TITLE
update docs about `store` in serializer

### PR DESCRIPTION
**Pull request by oriSomething**
related to #2841

### CODE REVIEW

**Comment by fivetanley** [comment]
I believe the store property here is public, but otherwise looks great!

File: packages/ember-data/lib/system/serializer.js [hunk 1]
```diff
@@ -23,6 +23,16 @@
...
 var Serializer = Ember.Object.extend({
+  /**
+    The `store` property is the application's `store` that contains all records.
+    It's injected as a service.
+    It can be used to push records from a non flat data structure server
+    response.
+
+    @property store
+    @type {DS.Store}
+    @private
```
Comment (stefanpenner) [inline packages/ember-data/lib/system/serializer.js]: @public

**Comment by oriSomething** [comment]
the `app veryor` returns weird message `node 0.10.37 x86 package not found`

**Comment by bmac** [comment]
I think the appveryor failing is unrelated to you changes. Do you mind squashing this pr down into 1 commit?

**Comment by oriSomething** [comment]
there is an option i can do it some way without forking again? or i need to fork it again?
(i don't mind…)

**Comment by bmac** [comment]
Here is what I would do @oriSomething 


First reset to before your merge commit (This will make the next steps easier).
`git reset --soft 4f8991864c8e9`

Then start a interactive rebase from your first commit.
`git rebase -i 386d5432~`

This command will bring you your editor in a screen that looks like this:
screen shot 2015-03-12 at 10 15 04 am

You will want to change `pick` on the second line to be `fixup`
screen shot 2015-03-12 at 10 17 09 am

Once you save and close your editor, git will merge both commits into 1. 

You can then force push back to your branch and this pr will be updated.
`git push -f oriSomething master`

**Comment by oriSomething** [comment]
awesome! thanks! i'll do that today

**Comment by bmac** [comment]
Thanks!

**Pull request merged**